lib/reset_password/new_password_success.dart

```diff
@@ -0,0 +1,164 @@
+    final theme = Theme.of(context);
+    // final style = theme.textTheme;
+    // final colors = theme.colorScheme;
+
+    return LayoutBuilder(
+      builder: (context, cts) {
+        return Scaffold(
+          resizeToAvoidBottomInset: false,
+          body: Stack(
+            children: [
+              BubblesTopBackGround(
+                cts: cts,
+                svgName: SvgNames.authBackground,
+              ),
+              Align(
+                alignment: Alignment.bottomCenter,
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  children: [
+                    SizedBox(
+                      height: 0.025 * cts.maxHeight,
+                    ),
+                    Stack(
+                      children: [
+                        Container(
+                          width: 99,
+                          height: 85,
+                          margin: EdgeInsets.all(30),
+                          decoration: BoxDecoration(
+                            color: theme.colorScheme.primary,
+                            borderRadius:
+                                BorderRadius.all(Radius.circular(4.0)),
+                          ),
+                        ),
+                        Positioned(
+                          bottom: 50,
+                          left: 55,
+                          child: ClipRRect(
+                            borderRadius: BorderRadius.circular(4.0), //or 15.0
+                            child: SvgPicture.asset(
+                              'assets/FpbIcons/blue_circle_check_mark.svg',
+                              width: 45.83,
+                              height: 45.83,
+                              colorFilter: ColorFilter.mode(theme.colorScheme.onPrimaryContainer, BlendMode.srcIn)
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                    SizedBox(
+                      height: 0.001 * cts.maxHeight,
+                    ),
+                    PageTitle(
+                      title: l10n.newPasswordSuccessTitle,
+                      box: cts,
+                    ),
+                    SizedBox(
+                      height: 0.02 * cts.maxHeight,
+                    ),
+                    Text(
+                      l10n.newPasswordSuccessBodyContent,
+                      textAlign: TextAlign.center,
+                      style: TextStyle(height: 1.5, fontSize: 12),
+                    ),
+                    SizedBox(
+                      height: 0.025 * cts.maxHeight,
+                    ),
+                    SizedBox(
+                      height: 0.03 * cts.maxHeight,
+                    ),
+                    FpbButton(
+                      label: l10n.signInLogInButtonLabel,
+                      onTap: () {
+                        context.router.push((SaveMoneyRoute()));
```

Comment:
@lkmandy Please bring the user back to the Login screen as the button text says. or send him to the home screen and update the button text accordingly.

Reply:
So from the Figma design, it's supposed to redirect the user to the screen below.
The Login button leads directly to an in-app screen rather than the Login screen

It's not been designed yet, I had put this SaveMoneyRoute alternative because it's the closest alternative that can serve as a placeholder for now

<img width="307" alt="Screen Shot 2023-04-05 at 12 28 49" src="https://user-images.githubusercontent.com/17765231/230067615-f678fe4f-8ee8-44af-8984-6a51ac1e9b98.png">